src/networks.json

```diff
@@ -980,6 +980,21 @@
     "start": 0,
     "logo": "ipfs://QmbENgHTymTUUArX5MZ2XXH69WGenirU3oamkRD448hYdz"
   },
+  "2000": {
+    "key": "2000",
+    "name": "Doge Chain",
+    "shortName": "dogechain",
+    "chainId": 2000,
+    "network": "mainnet",
+    "testnet": false,
+    "multicall": "0x6f9D3f3932B417bd4957585D236Cbc32b32C0BDc",
+    "rpc": [
+      "https://rpc.dogechain.dog"
+    ],
+    "explorer": "https://explorer.dogechain.dog",
+    "start": 0,
+    "logo": "https://raw.githubusercontent.com/dogechain-lab/docs/main/symbol_var_6.png"
```

Comment:
Can upload and pin it to IPFS plz :) To maintain consistency with other networks

Reply:
995 "logo": "ipfs://bafkreigovfh3pinsdih777issfgaflwu2yjzroljs2642gbvwikcd3nm4i
"